talk/owt/sdk/p2p/p2ppeerconnectionchannel.cc

```diff
@@ -920,117 +913,111 @@ void P2PPeerConnectionChannel::Stop(
   switch (session_state_) {
     case kSessionStateConnecting:
     case kSessionStateConnected:
-      SendStop(nullptr, nullptr);
```

Comment:
Why are these next two blocks being removed?

Reply:
The only difference between the three blocks was the first one was calling ClosePeerConnection(). But we want to close the connection even if the connection never reached connected state, otherwise the tracks on the underlying peer connection object will leak SHM clients. Now that ClosePeerConnection() is guarded with a lock, and is a no-op if already called once, it's safe to always call ClosePeerConnection() in Stop().